beacon-chain/p2p/discovery.go

```diff
@@ -182,6 +173,15 @@ func (s *Service) filterPeer(node *enode.Node) bool {
 	return true
 }
 
+// check if peer is at max limit.
```

Comment:
Can you please add a bit more to this? What limit? What peer?

Reply:
added comment